app/cluster.go

```diff
@@ -30,3 +30,117 @@ func (s *Server) InvokeClusterLeaderChangedListeners() {
 		})
 	})
 }
+
+func (a *App) Publish(message *model.WebSocketEvent) {
```

Comment:
This is not strictly cluster relate, this is related to sending thing through the websockets, and, because we support clustering, if we have a cluster configured, we also propagate that event to the cluster.

Maybe we can separate it and rename it a bit, something like "PublishMessageInLocalHub" and then create other for the "PublishMessageInCluster" or something like that.

Anyway, I think moving this methods here is not appropriate because is not the cluster code responsability, is more in the area of the hub responsability. IMO.

Reply:
Alright, no worries. I'll bring those back in.

Reply:
done